service/ztests/curl-cors.yaml

```
@@ -0,0 +1,51 @@
+script: |
+  source service.sh
+  echo "=== OPTIONS: allowed ==="
```

We don't quotes this stuff in other ztests.
```suggestion
  echo === OPTIONS: allowed ===
```